config-provisioning/src/main/resources/configdefinitions/node-repository.def

```diff
@@ -5,4 +5,6 @@ namespace=config.provisioning
 # version. Example: my-docker-registry.domain.tld:8080/dist/vespa
 dockerImage string default="dummyImage"
 
+serverNodeType enum {config, controller} default=config
```

Comment:
Oh, this is what you meant. Didn't know config definitions supported enums, great!